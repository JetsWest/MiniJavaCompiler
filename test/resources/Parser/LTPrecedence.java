class A {
    public static void main(String[] args) {
        System.out.println(new B().LT());
    }

}
class B {
    public int LT() {
        int a;
        int b;
        int c;
        a = 4;
        b = 5;

        if (a < b) {
            c = a;
        } else {
            c = b;
        }
        return c;
    }
}