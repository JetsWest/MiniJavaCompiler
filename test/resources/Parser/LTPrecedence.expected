Program
  MainClass A
    Print
      new B().LT()
  Class B
    MethodDecl LT
      returns int
      int a
      int b
      int c
      a = 4
      b = 5
      if (a < b)
        c = a
      else
        c = b
      Return c